packages/flutter_tools/lib/src/commands/build_aar.dart

```diff
@@ -33,6 +34,13 @@ class BuildAarCommand extends BuildSubCommand {
   @override
   final String name = 'aar';
 
+  @override
+  Future<Set<DevelopmentArtifact>> get requiredArtifacts async => <DevelopmentArtifact>{
+    DevelopmentArtifact.androidGenSnapshot,
+    DevelopmentArtifact.androidMaven,
```

Comment:
does this mean that the tool downloads the maven artifacts?  The tool might download more than what it's actually needed to build the artifacts.

Reply:
Ahh, will Gradle download these on its own then?

Reply:
yep.